lms/djangoapps/learner_dashboard/views.py

```diff
@@ -66,6 +67,7 @@ def program_details(request, program_uuid):
 
     context = {
         'program_data': program_data,
+        'courses_progress': meter.progress(programs=meter.programs, count_only=False)[0],
```

Comment:
are you suggesting we just delete the course data from the object if it is in fact unused?